lib/client/api_confluence.es6.js

```diff
@@ -27,39 +27,37 @@ foam.CLASS({
     {
       name: 'apiVelocityDAO',
```

Comment:
Here and elsewhere: I think we were barking up the wrong tree by correcting use of `typeName` here. We don't want `typeName`, we want to declare the type of the property. I think we want:

```js
{
  class: 'FObjectProperty',
  of: 'foam.dao.DAO',
  // ...
}
```